src/lcode.c

```diff
@@ -7,6 +7,10 @@
 #define lcode_c
 #define LUA_CORE
 
+#ifdef WITH_YK
```

Comment:
Is `WITH_blah` the standard Lua source idiom for "feature blah is enabled"?

Reply:
Maybe it should be `-DLUA_USE_YK` do be like the other stuff.

(Lua's existing `USE_*` can only be configured by editting the `Makefile`)

Reply:
Fixed in 37bbd58166